src/main/java/de/bonndan/nivio/assessment/AssessmentController.java

```diff
@@ -31,9 +28,7 @@ public AssessmentController(LandscapeRepository landscapeRepository, KPIFactory
             return ResponseEntity.notFound().build();
         }
 
-        Assessment assessment = new Assessment(
-                landscape.applyKPIs(factory.getConfiguredKPIs(landscape))
-        );
+        Assessment assessment = new Assessment(landscape.applyKPIs(landscape.getKpis()));
```

Comment:
This looks a bit strange at the first sight. I checked more deeply and realized that `applyKPIs()` is a default method in the `Asseable` interface that recursively calls itself. 

I would have suggested to make `applyKPIs()` in `Landscape` use the KPIs already known to the `Landscape`. I did not see any usage of `landscape.applyKPIs(Map<String, KPI> kpis)` where we would provide the KPIs from the outside.

Reply:
I think we could check how the `Landscape` object works after https://github.com/dedica-team/nivio/issues/212 has been resolved. Maybe thinks like this are already cleaned up after these changes.